torch/storage.py

```diff
@@ -1,11 +1,13 @@
 # mypy: allow-untyped-defs
```

Comment:
We should type all methods and remove this comment.

Reply:
Totally agree - but that's a separate (and much larger) change (which I'm doing in a background task).  This is just removing some stuff that mypy actively complains about.